README.md

```diff
@@ -442,27 +445,31 @@ Key   | Mode | Action
 
 Key   | Mode | Action
 ----- |:----:| ------------------
-`;`+`r` | Normal | Resumes Unite window
-`;`+`f` | Normal | Opens Unite file recursive search
-`;`+`i` | Normal | Opens Unite git file search
-`;`+`g` | Normal | Opens Unite grep with ag (the_silver_searcher)
-`;`+`u` | Normal | Opens Unite source
-`;`+`t` | Normal | Opens Unite tag
-`;`+`T` | Normal | Opens Unite tag/include
-`;`+`l` | Normal | Opens Unite location list
-`;`+`q` | Normal | Opens Unite quick fix
-`;`+`e` | Normal | Opens Unite register
-`;`+`j` | Normal | Opens Unite jump, change
-`;`+`h` | Normal | Opens Unite history/yank
-`;`+`s` | Normal | Opens Unite session
-`;`+`o` | Normal | Opens Unite outline
-`;`+`ma` | Normal | Opens Unite mapping
-`;`+`me` | Normal | Opens Unite output messages
-`<leader>`+`b` | Normal | Opens Unite buffers, mru, bookmark
+`[unite]` | Normal | unite leader, default is `f`, `:h g:spacevim_unite_leader`
+`[unite]`+`r` | Normal | Resumes Unite window
+`[unite]`+`f` | Normal | Opens Unite file recursive search
+`[unite]`+`i` | Normal | Opens Unite git file search
+`[unite]`+`g` | Normal | Opens Unite grep with ag (the_silver_searcher)
+`[unite]`+`u` | Normal | Opens Unite source
+`[unite]`+`t` | Normal | Opens Unite tag
+`[unite]`+`T` | Normal | Opens Unite tag/include
+`[unite]`+`l` | Normal | Opens Unite location list
```

Comment:
`fl` will open location list, but there should be no items in it, if you buffer have location list, whitch used by some plugin, such as neomake or ag.vim, it will work.